data/src/test/java/org/apache/iceberg/io/TestWriterMetrics.java

```diff
@@ -152,9 +152,8 @@ public void testPositionDeleteMetrics() throws IOException {
         writerFactory.newPositionDeleteWriter(outputFile, table.spec(), null);
 
     try {
-      T deletedRow = toRow(3, "3", true, 3L);
       PositionDelete<T> positionDelete = PositionDelete.create();
-      positionDelete.set("File A", 1, deletedRow);
+      positionDelete.set("File A", 1);
```

Comment:
The deprecation is this override of the `set` method, right? Anything else we would deprecate?